README.md

```diff
@@ -18,28 +18,28 @@ PostHog provides open source product analytics, built for developers. Automate t
 
 <img align="left" src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/rebrand/icons/Untitled_Artwork+3.jpg" width="50px" />
 
-## In-depth Walkthrough 
+## Make the Best of PostHog
 
-See the [PostHog Docs](https://posthog.com/docs/) for in-depth walk throughs on functionality.
+See [PostHog Docs](https://posthog.com/docs/) for in-depth walk-throughs on functionality.
```

Comment:
Nit: "walk throughs" was definitely wrong, and while "walk-throughs" is correct, I personally prefer the alternative spelling "walkthroughs"

Reply:
Really doesn't matter though, I've kept it in.